leasing/management/commands/attach_areas.py

```diff
@@ -108,14 +119,15 @@ def handle(self, *args, **options):  # noqa: C901 TODO
                         )
                         intersection.transform(3879)
                         if intersection.area < 1:
-                            self.stdout.write(
-                                "Lease #{} {}: DISCARD area {}: intersection area too small".format(
-                                    lease.id, lease.identifier, intersect_area.id
-                                )
+                            LOG.debug(
+                                "Lease #%s %s: DISCARD area %s: intersection area too small",
+                                lease.id,
+                                lease.identifier,
+                                intersect_area.id,
                             )
                             continue
                     except GEOSException as e:
-                        self.stdout.write(str(e))
+                        LOG.exception(str(e))
```

Comment:
Here's a similar case than the previous `LOG.exception`. Do the same here...

Reply:
Changed